.github/workflows/preview-prepare.yml

```diff
@@ -12,6 +12,7 @@ on:
       - 'core/test/**'
       - 'loader-tests/**'
       - '.devcontainer/**'
+      - 'extensions/**'
```

Comment:
We try to avoid calling CI without a reason.

For instance, we do not publish staging on changes irrelevant to web client.